### PR TITLE
Comprehensive documentation 

### PR DESCRIPTION
**Pull request by srossross**
### Checklist:

* [x] The readme and documentation has been updated or does not need to be updated
* [ ] This change has been communicated with the team if needed
* [ ] I have linked to the original issue
* [ ] I have added tests to cover my changes
* [x] This PR does not need notes or I've added them below
* [x] This PR does not need extra testing from our release testing process or I've added a 'tests' section below

### CODE REVIEW

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
-4. Click **Close**.
+1. Setup your rasbperry as explained in the [secton below](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#setting-up-the-rasbperry-pi) This step is only required once
+2. Before you want to connect the application, you must run the emulator on the pi [emulator](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#install-an-run-the-base-station-emulator)
+3. To test that the emulator is working you can use a mobile tool like:
+    * [LightBlue](https://punchthrough.com/lightblue/) for mobile
+    * [BlueSee](https://www.synapse.com/bluesee) for mac desktop
+4. You should be able to get and set the GATT characteristics as documented in our [API documentation](https://endpointsportal.plantiga-dev.cloud.goog/docs/api.eyeofthetiga.com/1/c/Guides/BaseStation)
 
-### Enable the API
+## Seup Flow
```
Comment (ahollist) [inline api.eyeofthetiga.com/Guides/Getting Started.md]:
```suggestion
## Setup Flow 
```

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 2]
```diff
@@ -44,10 +48,13 @@ The Cloud Adapter Emulator is intended to be run on a Raspberry Pi running the [
 5. Plug in the microSD card to the Raspberry Pi and plug in the power, allowing up to 5 minutes for the Pi to boot for the first time. The default username is: `pi` and its password is: `raspberry`.
    1. If you are logging in via the tty using Keyboard and HDMI, use `wpa_cli` to configure the wifi, create the file from `step 4` in the location `/etc/wpa_supplicant/wpa_supplicant.conf` and reboot or restart the `wpa_supplicant.service`, or use an ethernet cable to connect to the internet.
    2. If you used `step 3` and `step 4` to set up your Pi, you can connect via ssh (if you are on the same network) using the hostname: `raspberrypi.local`.
-6. Download the latest version of the Emulator binary using the command 
+
+## Install an run the Base Station Emulator
```
Comment (ahollist) [inline api.eyeofthetiga.com/Guides/Getting Started.md]:
```suggestion
## Install and run the Base Station Emulator
```

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 3]
```diff
@@ -1,31 +1,35 @@
...
-2. [Enable billing](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project) for your project.
+## Development Flow 
 
-### Creating an API key
-1. [Create an API key](https://console.developers.google.com/apis/credentials) in the Google APIs Console.
-2. Click **Create credentials**, then select **API key**.
-3. Copy the key to the clipboard.
-4. Click **Close**.
+1. Setup your rasbperry as explained in the [secton below](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#setting-up-the-rasbperry-pi) This step is only required once
+2. Before you want to connect the application, you must run the emulator on the pi [emulator](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#install-an-run-the-base-station-emulator)
```
Comment (ahollist) [inline api.eyeofthetiga.com/Guides/Getting Started.md]:
```suggestion
2. Before you want to connect the application, you must [run the emulator on the pi](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#install-and-run-the-base-station-emulator)
```

**Review by ahollist — CHANGES_REQUESTED** [review]
just a couple typos to update

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 4]
```diff
@@ -1,31 +1,35 @@
...
-1. Create a new [Cloud Platform project](https://console.developers.google.com/projectcreate).
-2. [Enable billing](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project) for your project.
+## Development Flow 
 
-### Creating an API key
-1. [Create an API key](https://console.developers.google.com/apis/credentials) in the Google APIs Console.
-2. Click **Create credentials**, then select **API key**.
-3. Copy the key to the clipboard.
-4. Click **Close**.
+1. Setup your rasbperry as explained in the [secton below](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#setting-up-the-rasbperry-pi) This step is only required once
```
Comment (ahollist) [inline api.eyeofthetiga.com/Guides/Getting Started.md]:
```suggestion
1. Setup your Rasbperry Pi as explained in the [secton below](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#setting-up-the-rasbperry-pi). This step is only required once
```

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 5]
```diff
@@ -1,31 +1,35 @@
...
-### Setting up the Cloud Adapter Emulator
+### 2. API SERVICE `0xD100`
 
+1. Get the API key by exchanging a `firebase id token` with a `cloud-adapter-token` using our API [/cloud-gears/cloud-adapter-token](https://endpointsportal.plantiga-dev.cloud.goog/docs/api.eyeofthetiga.com/1/routes/cloud-gears/cloud-adapter-token/get)
+1. Set the API key `0xD101` 
+1. Check the API status `0xD102` for `KEY_OK`
 
-The Cloud Adapter Emulator is intended to be run on a Raspberry Pi running the [Raspberry Pi OS Lite](https://www.raspberrypi.org/software/operating-systems/).
+
+## Setting up the Rasbperry PI
```
Comment (ahollist) [inline api.eyeofthetiga.com/Guides/Getting Started.md]:
```suggestion
## Setting up the Rasbperry Pi 
```

**Review by ahollist — APPROVED** [review]
just some small edits but otherwise lgtm

File: api.eyeofthetiga.com/Guides/Getting Started.md [hunk 6]
```diff
@@ -1,31 +1,35 @@
...
-2. [Enable billing](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project) for your project.
+## Development Flow 
 
-### Creating an API key
-1. [Create an API key](https://console.developers.google.com/apis/credentials) in the Google APIs Console.
-2. Click **Create credentials**, then select **API key**.
-3. Copy the key to the clipboard.
-4. Click **Close**.
+1. Setup your Rasbperry Pi as explained in the [secton below](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#setting-up-the-rasbperry-pi). This step is only required once
+2. Before you want to connect the application, you must [run the emulator on the pi](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#install-and-run-the-base-station-emulator)
```
Comment (Rowansdabomb) [inline api.eyeofthetiga.com/Guides/Getting Started.md:6]:
```suggestion
2. Before you connect the application, you must [run the emulator on the pi](https://github.com/Plantiga/endpoints-developer-portal/blob/srossross-patch-2/api.eyeofthetiga.com/Guides/Getting%20Started.md#install-and-run-the-base-station-emulator)
```

**Review by Rowansdabomb — APPROVED** [review]

**Pull request merged**